pkg/rbac/role.go

```diff
@@ -72,6 +66,12 @@ func OrganizationClusterAdminPolicyRules() []rbacv1.PolicyRule {
 			APIGroups: []string{corev1.GroupName},
 			Resources: []string{"secrets"},
 		},
+		// Grant permission for TeamRoles
+		{
+			Verbs:     []string{"get", "list", "watch", "update", "patch", "delete", "create"},
+			APIGroups: []string{greenhouseapisv1alpha1.GroupVersion.Group},
+			Resources: []string{"teamroles"},
```

Comment:
Should org-admins not be able to manipulate all Greenhouse resources? So we could lose lines 17-21 of this file?
```suggestion
			Resources: []string{"*"},
```

Reply:
Ah no. We have to distinguish `organizations` only in orgs namespace...
But still we can consolidate with lines 17-21 and add clusters + plugins
And we should add the List/watch/get of PluginDefinitions to the org-admin as well, right?

Reply:
```suggestion
			Resources: []string{"clusters", "plugins", "pluginpresets", "teams", "teammemberships", "teamroles", "teamrolebindings"},
```